gnu-apl-plot.el

```diff
@@ -4,6 +4,12 @@
 (require 'gnu-apl-util)
 (require 'gnu-apl-network)
 
+(declare-function gnu-apl--name-at-point "gnu-apl-documentation" ())
+(declare-function gnu-apl--choose-variable "gnu-apl-editor"
+		  (prompt &optional type default-value))
+
+(defvar gnu-apl-gnuplot-program)	;gnu-apl-mode.el
```

Comment:
We can just `require` here.

Reply:
When you do that, there is a recursive require chain between gnu-apl-mode and gnu-apl-plot.

Reply:
Then I suggest we move the `defcustom` to `gnu-apl-plot.el` instead.

Reply:
Done!